tests/utils_test.py

```diff
@@ -0,0 +1,22 @@
+"""Unit tests for english language checker"""
+import unittest
+import libcipher.utils as utils
+
+filepath = '/home/travis/build/Chennaipy/libcipher/libcipher/dictionary.txt'
```

Comment:
@bravegnu  I am not sure if I understood what you said. I have locally modified utils_test.py to get the filepath and create file objects using
    filepath = os.path.dirname(os.path.abspath(inspect.getfile(utils)))
    fileobject = open(filepath+'/dictionary.txt')
I pass fileobject to utils.py
The tests have run fine. Is this what you meant by "changing the API to accept a file object instead of a filename" ?
Thank you